DATA/production/calib/tpc-pedestal.sh

```diff
@@ -21,26 +21,32 @@ HOST=localhost
 QC_CONFIG="components/qc/ANY/any/tpc-pedestal-calib-qcmn?run_type=${RUNTYPE:-}"
 CALIB_CONFIG="TPCCalibPedestal.ADCMin=20"
  
-
-max_events=50
-publish_after=400
-
-if [[ ! -z ${TPC_CALIB_MAX_EVENTS:-} ]]; then
-    max_events=${TPC_CALIB_MAX_EVENTS}
-fi
-
-if [[ ! -z ${TPC_CALIB_PUBLISH_AFTER:-} ]]; then
-    publish_after=${TPC_CALIB_PUBLISH_AFTER}
-fi
-
+# ===| configuration from environment variables |===============================
+max_events=${TPC_CALIB_MAX_EVENTS:-50}
+publish_after=${TPC_CALIB_PUBLISH_AFTER:-400}
+sendToDCS=${TPC_CALIB_SEND_TO_DCS:-1}
+
+# ===| ccdb populator setup |===================================================
+# the production CCDB populator will accept subspecs in this range
+CCDBPRO_SUBSPEC_MIN=0
+CCDBPRO_SUBSPEC_MAX=32767
+CCDBPATHPRO="http://o2-ccdb.internal"
+
+# the DCS CCDB populator will accept subspecs in this range
+CCDBDCS_SUBSPEC_MIN=32768
```

Comment:
Hello @wiechula , can you remind me if these numbers are assigned in a specific way? I mean, how did you determine them? This is probably not important since this is a pedestal run.

Reply:
To my understanding they are arbitrary. I used what the mean vertex task is using.
The only thing is that the subspecs used in the during `snapshop` need to match what is configured here.

Reply:
It is in the TPC standalone calib workflow, yes

Reply:
Thanks.